### PR TITLE
Added verified checkmark to packages

### PR DESCRIPTION
**Pull request by dcc7497**
* On the NuGet packages area of the Used By section, those packages that are verified will have the verified icon (white checkmark enclosed in a blue circle) next to their package name
 
image

Progress on https://github.com/NuGet/Engineering/issues/3221

### CODE REVIEW

File: src/NuGetGallery.Services/PackageManagement/PackageService.cs [hunk 1]
```diff
@@ -166,9 +166,9 @@ private IReadOnlyCollection<PackageDependent> GetListOfDependents(string id)
                                 join p in _entitiesContext.Packages on pd.PackageKey equals p.Key
                                 join pr in _entitiesContext.PackageRegistrations on p.PackageRegistrationKey equals pr.Key
                                 where p.IsLatestSemVer2 && pd.Id == id
-                                group 1 by new { pr.Id, pr.DownloadCount, p.Description } into ng
+                                group 1 by new { pr.Id, pr.DownloadCount, pr.IsVerified, p.Description } into ng
```
Comment (joelverhagen) [inline src/NuGetGallery.Services/PackageManagement/PackageService.cs:169]: Unit tests?

File: src/NuGetGallery.Services/Models/PackageDependent.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@ public class PackageDependent
         public string Id { get; set; }
         public int DownloadCount { get; set; }
         public string Description { get; set; }
+        public bool IsVerified { get; set; }
 
         // TODO Add verify checkmark
```
Comment (joelverhagen) [inline src/NuGetGallery.Services/Models/PackageDependent.cs:15]: We can drop the TODO now right? 👍

File: src/Bootstrap/dist/css/bootstrap-theme.css [hunk 3]
```diff
@@ -912,14 +912,6 @@ img.reserved-indicator-icon {
...
   color: red;
 }
-.page-package-details {
-  /*
-  .used-by-adjust-stars {
-    word-break: normal;
-    display: inline-block;
-  }
-      */
-}
```
Comment (dcc7497) [inline src/Bootstrap/dist/css/bootstrap-theme.css:922]: This commented code was unintentionally left in a previous PR, the UI PR. When grunt is ran it gets removed, as expected

File: src/Bootstrap/less/theme/page-display-package.less [hunk 4]
```diff
@@ -180,6 +180,13 @@
         font-weight: 400;
       }
     }
+
+    .reserved-indicator {
+      width: 14px;
+      margin-bottom: 3px;
+      margin-left: 2px;
+     vertical-align: middle;
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/page-display-package.less:188]: nit inconsistent indentation

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 5]
```diff
@@ -574,6 +574,13 @@
                                                 <a class="text-left ngp-link" href="@Url.Package(item.Id)">
                                                     @(item.Id)
                                                 </a>
+                                                @if (item.IsVerified)
+                                                {
+                                                    <img class="reserved-indicator"
+                                                         src="~/Content/gallery/img/reserved-indicator.svg"
+                                                         @ViewHelpers.ImageFallback(Url.Absolute("~/Content/gallery/img/reserved-indicator-25x25.png"))
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml:581]: Can you see if the PNG version looks okay? The image itself is 25x25 and will be scaled to 14x14 by the browser. It's possible it looks bad scaled down like this. If it does, we should create 14x14 version from the SVG and use that instead.

You can see the PNG if you break the `src` path (like change `.svg` to `.svgbroken`).

Also, when do we have `~` at the beginning of the path?

**Pull request merged**